lib/engine/config/game/g_1846.rb

```diff
@@ -20,9 +20,21 @@ module G1846
     "5": 9000
   },
   "certLimit": {
-    "3": 14,
-    "4": 12,
-    "5": 11
+    "3": {
```

Comment:
are these changes part of a different commit?

Reply:
Yeah, it's part of a commit that was already merged. I'll rebase and I think this should disappear.